src/common/utxobased/engine/makeUtxoEngineState.ts

```diff
@@ -792,12 +791,12 @@ export const pickNextTask = async (
   }
 
   // loop to get and process transaction history of single addresses, triggers setLookAhead
-  for (const address of Object.keys(transactionsCache)) {
-    const state = transactionsCache[address]
+  for (const address of Object.keys(addressTransactionCache)) {
```

Comment:
Sense we're switching to entries, might as well do it everywhere because we're touching these lines of code.